jupyter-extension/jupyterlab_pachyderm/pfs_manager.py

```diff
@@ -347,15 +319,19 @@ def get(
             )
 
         # Show an empty toplevel model if no branch is specified
-        if branch is None:
-            return self._get_toplevel_model(content=False, branch=branch)
+        if self.mounted_branch is None:
+            return self._get_content_model("", "/", DEFAULT_DATETIME, None)
 
-        file = self._get_file_from_path(path=path, branch=branch)
+        mounted_branch_name = _default_name(self.mounted_branch)
+        mounted_branch_created = self._client.pfs.inspect_commit(
```

Comment:
Is this code pulled from somewhere? I don't think there's any harm in it, but I don't believe we ever came to a decision on what `created` corresponds to in pachyderm model. I think its ok to leave this as the default if we are unsure

Reply:
I didn't pull it from anywhere directly, but I saw a few references to start_time and started which seemed correct? like in `src/internal/clusterstate/v2.7.0_test.go#152`

Reply:
Oh I didn't fully understand this when I looked at it the first time. This timestamp is only used when `file is None` right? I think this makes sense then. I thought this timestamp was being associated with all the files within this branch.

Reply:
Yep! Sounds good then